drivers/trim_util.py

```diff
@@ -104,15 +104,26 @@ def do_trim(session, args):
         time.sleep(LOCK_RETRY_INTERVAL)
 
     if got_lock:
-        try:
-            vg_name = _vg_by_sr_uuid(sr_uuid)
-            lv_name = sr_uuid + TRIM_LV_TAG
-            lv_path = _lvpath_by_vg_lv_name(vg_name, lv_name)
-
-            # Clean trim LV in case the previous trim attemp failed
-            if lvutil.exists(lv_path):
-                lvutil.remove(lv_path)
```

Comment:
Why have we moved the following block out of the try block? What if lvutil.remove throws an exception?